nservicebus/messaging/databus/azure-blob-storage_cleanup-setup_ABSDataBus_[1.0,4.0).partial.md

```diff
@@ -10,4 +10,62 @@ Review the [sample](/samples/azure/blob-storage-databus-cleanup-function/) to se
 
 ### Using the Blob Lifecycle Management policy
 
-Attachment blobs can be cleaned up using the [Blob Storage Lifecycle feature](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-lifecycle-management-concepts). This method allows configuring a single policy for all data bus-related blobs. Those blobs can be either deleted or archived. The policy does not require custom code and is deployed directly to the storage account. This feature can only be used on GPv2 and Blob storage accounts, not on GPv1 accounts. 
+Attachment blobs can be cleaned up using the [Blob Storage Lifecycle feature](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-lifecycle-management-concepts). This method allows configuring a single policy for all data bus-related blobs. Those blobs can be either deleted or archived. The policy does not require custom code and is deployed directly to the storage account. This feature can only be used on GPv2 and Blob storage accounts, not on GPv1 accounts.
+
```

Comment:
As we discussed we should tie the configuration of the policy to any configured values in the data bus. Here is a suggestion.

```suggestion

NOTE: The `prefixMatch` setting should be set to the value of `databus/` by default. If [the `Container()` or `BasePath()` configuration options](#behavior) have been specified when configuring the data bus the `prefixMatch` setting must modified to take into account the configured container and/or base path values.

```